src/Services/Temp/Api/Api.http

```diff
@@ -0,0 +1,6 @@
+@Api_HostAddress = http://localhost:5241
+
+GET {{Api_HostAddress}}/weatherforecast/
+Accept: application/json
+
+###
```

Comment:
Why do you need this? If it is for some kind of tests, then it should not fall into the reptoziria